src/playground/components/Configuration.js

```diff
@@ -109,6 +109,31 @@ export default function Configuration({ rulesMeta, eslintVersion, onUpdate, opti
     const ruleInputRef = useRef(null);
     const [rulesWithInvalidConfigs, setRulesWithInvalidConfigs] = useState(new Set([]));
 
+    const handleRuleChange = () => {
+        selectedRules.forEach(selectedRule => {
+            if (ruleNames.includes(selectedRule)) {
+                options.rules[selectedRule] = ["error"];
+                onUpdate(Object.assign({}, options));
+                ruleInputRef.current.setValue("");
+            }
+        });
+    };
+
+    const Input = props => {
+        if (props.isHidden) {
+            return <components.Input {...props} />;
+        }
+        return (
+            <components.Input {...props} onKeyDown={e => {
+                if ((e.metaKey || e.ctrlKey) && e.code === "Enter") {
+                    handleRuleChange();
+                } else {
+                    props.onKeyDown(e);
```

Comment:
this throwing error.
I think we cannot trigger a event like this. 
<img width="761" alt="image" src="https://user-images.githubusercontent.com/60533560/210965502-e0535297-1218-4d43-9043-dcff1e1ebfa3.png">

Reply:
Yea I'll remove that anyways its not needed